features/cellular/framework/targets/UBLOX/N2XX/UBLOX_N2XX_CellularSMS.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2017, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "UBLOX_N2XX_CellularSMS.h"
+
+using namespace mbed;
+
+UBLOX_N2XX_CellularSMS::UBLOX_N2XX_CellularSMS(ATHandler &atHandler) : AT_CellularSMS(atHandler)
+{
+}
+
+UBLOX_N2XX_CellularSMS::~UBLOX_N2XX_CellularSMS()
+{
+}
+
+nsapi_error_t UBLOX_N2XX_CellularSMS::set_cpms(const char *memr, const char *memw, const char *mems)
+{
+    return NSAPI_ERROR_UNSUPPORTED;
```

Comment:
Do we want to use the is_supported() type method of checking (as we've used this before in other classes above.)

Reply:
I have overridden virtual function, like this, which are unsupported and for others added in unsupported So that minimum changes shall be made in ARM classes.